omnigibson/__init__.py

```diff
@@ -97,7 +98,34 @@ def print_save_usd_warning(_):
 def create_app():
     global app
     from omni.isaac.kit import SimulationApp
-    app = SimulationApp({"headless": gm.HEADLESS})
+    remote_rendering = bool(os.getenv("REMOTE_ENABLED"))
+    app = SimulationApp({"headless": gm.HEADLESS or remote_rendering})
+
+    # Default Livestream settings
+    if remote_rendering:
+        app.set_setting("/app/window/drawMouse", True)
+        app.set_setting("/app/livestream/proto", "ws")
+        app.set_setting("/app/livestream/websocket/framerate_limit", 120)
+        app.set_setting("/ngx/enabled", False)
+
+        from omni.isaac.core.utils.extensions import enable_extension
+
+        # Note: Only one livestream extension can be enabled at a time
+        # Enable Native Livestream extension
+        # Default App: Streaming Client from the Omniverse Launcher
+        # enable_extension("omni.kit.livestream.native")
+
+        # Enable WebSocket Livestream extension
+        # Default URL: http://localhost:8211/streaming/client/
+        # enable_extension("omni.services.streamclient.websocket")
```

Comment:
when would these other options be used?

Reply:
I mostly left the code unmodified from `standalone_examples/api/omni.isaac.kit/livestream.py` to ease diff'ing in the future if nvidia changes something. From my testing, WebRTC is the best for low-latency connections, websocket is a little more robust for constrained bandwidth, and native (mirror omni on a local omni) has little use since it essentially limits you to a windows/linux computer, and in our situation we are usually only on a windows/linux computer if we are testing something.